bootstrap_and_teardown.md

```diff
@@ -0,0 +1,93 @@
+# Bootstrap procedure
```

Comment:
This file was just renamed, yeah?

Reply:
This file was extracted from the original `README.md`